docs/index.md

```diff
@@ -4,3 +4,7 @@
 
 This guide is describes the requirements to be fulfilled by any FIWARE Generic
 Enabler (GE) as well as the recommended practices for its development.
+
+The
```

Comment:
You could also just copy the same text verbatim. That way the reader would get all the information in styled CSS rather than being throw over to GitHub when clicking  a link